src/Queue/Connectors/RabbitMQConnector.php

```diff
@@ -42,7 +43,15 @@ public function connect(array $config): Queue
         if (false === class_exists($factoryClass) || false === (new \ReflectionClass($factoryClass))->implementsInterface(InteropAmqpConnectionFactory::class)) {
             throw new \LogicException(sprintf('The factory_class option has to be valid class that implements "%s"', InteropAmqpConnectionFactory::class));
         }
-
+        
+        $delayStrategyClass = RabbitMqDlxDelayStrategy::class;
+        if (array_key_exists('delay_strategy_class', $config)) {
```

Comment:
Since we are requiring php 7.1, you can use 
`$delayStrategyC = $config['delay_strategy'] ?? RabbitMqDlxDelayStrategy::class;